Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/Conversation.kt

```diff
@@ -224,22 +233,41 @@ fun Messages(
                     isLastMessageByAuthor = isLastMessageByAuthor
                 )
             }
+            // Add some space for the top app bar
+            item {
+                // TODO: Get height from somewhere.
+                Spacer(
+                    modifier = Modifier
+                        .height(90.dp)
```

Comment:
It's the height of the top app bar which comes from content height + paddings so it's hard to extrapolate. Mocks would be clearly 90.dp so that would also be a source.